doc/api/deprecations.md

```diff
@@ -818,6 +818,15 @@ a future version at which point only authentication tag lengths of 128, 120,
 is not included in this list will be considered invalid in compliance with
 [NIST SP 800-38D][].
 
+<a id="DEP00XX"></a>
+### DEP00XX: crypto.DEFAULT_ENCODING
+
+Type: Documentation-only
+
+The [`crypto.DEFAULT_ENCODING`][] property has been deprecated and replaced
+with `crypto.setDefaultEncoding()` and `crypto.getDefaultEncoding()`
+alternatives.
```

Comment:
Can we keep this wording consistent with other messages by changing it to this?:

> The [`crypto.DEFAULT_ENCODING`][] property is deprecated. Please use `crypto.getDefaultEncoding()` and `crypto.setDefaultEncoding()` instead.